frontend/src/components/worksheets/BundleDetail/BundleDetail.jsx

```diff
@@ -174,9 +174,8 @@ class BundleDetail extends React.Component<
                         }
                     }.bind(this),
                 );
-                $.when.apply($, fetchRequests).then(() => {

```

Comment:
Similar question here

Reply:
https://medium.com/sungthecoder/making-multiple-ajax-calls-and-deciphering-when-apply-array-b35d1b4b1f50
So here we're doing multiple AJAX calls and only go on to the then command if all of them are resolved.